rest-tests/src/test/java/io/syndesis/qe/rest/tests/steps/connection/Connections.java

```diff
@@ -89,7 +89,7 @@ public void createConnection(DataTable connectionProperties) {
     @Given("^create ActiveMQ connection$")
     public void createActiveMQConnection() {
         final List<List<String>> table = new ArrayList<>();
-        final Account account = accountsDirectory.getAccount("AMQ").get();
+        final Account account = accountsDirectory.getAccount(Account.Name.ACTIVEMQ.toString()).get();
```

Comment:
Can we add a `getAccount(Account.Name)` method to `AccountsDirectory`?

Reply:
:+1: